src/client/js/data/fetching.js

```diff
@@ -0,0 +1,22 @@
+export async function fetchData(filter, page) {
```

Comment:
Given that babel is being used, won't the usage of `async`/`await` require babel's regenerator runtime? I believe that's roughly 6 kB of overhead

Reply:
Infact I added it in the `webpack.config.js`:)

Reply:
So you want to add the regenerator runtime? My point was that it will bloat the bundle so I would rather not have it

Reply:
Since we agreed on not support IE11, will remove it.